tutorials/building_a_simple_web_blog_with_vweb/README.md

```diff
@@ -262,11 +250,14 @@ pub fn (app &App) find_all_articles() []Article {
 }
 ```
 
+Notice that the `Article` structure conforms to the same structure and naming as 
+the database table in the creation SQL statement. Also we need to add ORM decorators
+to our primary key to let it know that it is the primary key and it should auto-increment
+
 Let's fetch the articles in the `index()` action:
 
 ```v ignore
-// blog.v
-pub fn (app &App) index() vweb.Result {
+pub fn (app App) index() vweb.Result {
```

Comment:
New to add `[/index]` route attribute. A change is going in that will make you have to have explicit route attributes for the endpoint to be exposed.

Reply:
Updated in README.md and the source code.